README.md

```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
 `$ jfrog plugin uninstall jfrog-support-bundle-flunky`
 
 ## Usage
-### Commands TBD
-* jfrog-support-bundle-flunky
-    - Arguments:
-        - case - The JFrog Support case number.
-    - Example:
+
+This plugin has a unique command `support-bundle` that:
+1. Creates a Support Bundle on the target Artifactory service
+2. Downloads the Support Bundle locally in a temporary file
+3. Uploads the Support Bundle on JFrog "dropbox" service
+
+**Arguments**
+- `case` - The JFrog Support case number (required).
+
+**Example**
 ```
-$ jfrog jfrog-support-bundle-flunky 1234
+$ jfrog support-bundle 1234
```

Comment:
We should add an alias `sb` example too.

Reply:
We have it, I just don't know if it works...